src/styles/createTypography.d.ts

```diff
@@ -36,7 +37,7 @@ export interface TypographyStyle {
 
 export type Typography = { [type in Style]: TypographyStyle } & FontStyle;
 
-export type TypographyOptions = Partial<FontStyle> & Partial<Typography>;
+export type TypographyOptions = Partial<FontStyle> | DeepPartial<TypographyStyle>;
```

Comment:
Should be just

```ts
export type TypographyOptions = DeepPartial<Typography>;
```

Reply:
It doesn't work like that

Reply:
What's the problem?

Reply:
Trying to reproduce it on the playground. No luck so far. Maybe it's a problem with my local env